examples/videocomposer/vc/engine/modules/decoder.py

```diff
@@ -0,0 +1,36 @@
+import mindspore.nn as nn
+import mindspore.ops as ops
+from mindspore import Tensor
+
+
+class Decoder(nn.Cell):
```

Comment:
add short comments like "decode latent frames to pixel frames"? decoder can be a too general name.

Reply:
done